Sources/ComposableArchitecture/Internal/Deprecations.swift

```diff
@@ -5,6 +5,13 @@ import XCTestDynamicOverlay
 
 // MARK: - Deprecated after 0.49.2
 
+@available(
+  *,
+  deprecated,
+  message: "Use 'ReducerBuilder<_, _>' with explicit 'State' and 'Action' generics, instead."
+)
```

Comment:
curious what the downside of using this typealias, as I use it almost in every feature.